### ISSUE TITLE
ColorWriteMask All value is incorrect

### ISSUE THREAD

**Issue by bnason-nf** [eliemichel/WebGPU-Cpp#19]
```
ENUM(ColorWriteMask)
	ENUM_ENTRY(None, 0x00000000)
	ENUM_ENTRY(Red, 0x00000001)
	ENUM_ENTRY(Green, 0x00000002)
	ENUM_ENTRY(Blue, 0x00000004)
	ENUM_ENTRY(Alpha, 0x00000008)
	ENUM_ENTRY(All, WGPUColorWriteMask_None)
	ENUM_ENTRY(Force32, 0x7FFFFFFF)
END
```

It should instead be `0xF` or `Red | Green | Blue | Alpha`.

**Comment by bnason-nf** [comment]
Oh probably `WGPUColorWriteMask_All` is more consistent.

**Comment by eliemichel** [comment]
Fixed in [562d3ae](https://github.com/eliemichel/WebGPU-Cpp/commit/562d3aef761ee6a22ffbe363a9e468296a66d9e6)!
